inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -1149,3 +1149,27 @@ def load_record_from_hep(obj, wf):
     obj.extra_data['head_revision_id'] = record_data['revision_id']
 
     return obj
+
+
+def remove_inspire_categories_derived_from_core_arxiv_categories(obj, eng):
+    if not obj.data.get("arxiv_eprints"):
+        return
+    inspire_categories_without_arxiv_sourced = [
+        category
+        for category in obj.data.get("inspire_categories")
```

Comment:
```suggestion
        for category in obj.data.get("inspire_categories", [])
```

Reply:
I thought that `inspire_categories` are always present when `arxiv_eprints` are in the record, but you are right, it might happen that there's an `arxiv_eprint` without a category 👍